### PR TITLE
seed yukon Hub cluster

### PR DESCRIPTION
**Pull request by novacain1**
Seeds the Yukon cluster into the vse-carslab-hub repository.  This cluster is to be used for productive work in the BOS2 Red Hat partner laboratory.  Note that this cluster *may* move to its own repository at a later date, seeding here for now.

Also adjusted some mis-spellings, product nomenclature updates, and consistency in a few of the READMEs.

### CODE REVIEW

File: clusters/overlays/yukon-cars2/patches/pullsecret-sealed-patch.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
...C9BqeAeE2vQkA5GRIZ1zsmHCbKaHCS3GlKzZqORozHMY4QAffeJjhAtqf/WyW3IFpd1Fo3emAMZSPtbEGSrvjz4DQE1hdtkNGGy/g1zDPljjeAPPLDtbEEam7SAbWk3bseKCmxCYMNfMACFoUsPd/w+QHSH2YgHQvZQwpdzHClcdCexBRAajOaXo67ijTampUz+18HNYn/2OHjYP03BJppw+aCW/716wRoKGP372y+b5m2yspUsl/5SN6zhZ6q+Iuzm6GNfqpiN04/AWDPFo6ZWCdspqNX4+f3P9ZyTeGy8nMGCCrNPRmJF/odmoWlPNtLc2rOH2KpMi39tQTv1r5ojTH3IeDFPFxZgMQUwEyyFxPYPzDdl1AZUqgP3lJxUDssnnzbpkNGguo9fzTziBWWrqkXhZrB0Du/ISP60zxLxyqzqWYENqByRQDfwLZg7JJI9gLfmIb87etfabRvG4j/dVZeaGsuWr7xpZSOAwa92FOiRZVn+KBAFyV0lSiTXo0nQRtWKDRxZFQGf3XMOXSczQ3xm8T/Eon84jAX3N7LQORb7+oEu3kd/yPI0yhE9Z7XWSfFmSm9tBaji2bocO3AgeBalEN46DIDwGGn6fwlS0bPYgngupECeg+iPJM7K3FHCMx6wVaLbHZaVC3P9SZ9zjEu5w+kZXp2FGM61qSHyiLHyP0H4963PKHyp2VCMFJZLuotN+BocPa+0oESY6UCVK5vWnzUJOIKmv5rS16mlY1P7x3/mXvVJ0B9F5TYW7WbXI9XNfXouoK/08II4odgyWFcdvBigF30VLh7Kza3am9wzpg+uTM0WTGNtOHcVikrUmagAtqJM+o9eDtDW1EIprn/FXX8vIV7bJ1sZgTWvNElIRS3Bnmb0+2ddFs8w9tyjKOufC/B48lIFep15XPyCz/0alGhJFpHEtaJerhIR7XSBbe3/KabOf75UNgLCnCe5mOvFuREuVHfPhPtdKWX1ApTOlzmzhZUw5RrgDJaRNgrWosLJMlgDFJgZF1fpUG9j6GRn0JJltB1gJPewDU5Hfh8rFN+MlCYQwZXQSNsiJVusTljcq7r72bAcfUiWiHMfcCf28xaPkjhbKoPKFCyqQMmXZE49GAjPre4Bv6sn6oOWgb94QUF5fpEWeD7j4yaVW78P4gapTY4VmW/2lxmX72sVxPqGCQzlB6zgKKOeyfQ='
```
Comment (natifridman) [inline clusters/overlays/yukon-cars2/patches/pullsecret-sealed-patch.yaml:15]: Is this your sealed pull secret? if this is something you created in another cluster you'll need to import the sealed secret key to this cluster or re-encrypt it

**Review by natifridman — COMMENTED** [review]
Looks good, thanks for the doc fixes.
There are some yamllint warnings in the nmstate file that I think we can fix.

I think that eventually, this should go to the `vse-clusters` repo so this repo can stay clean and serve us in multiple environments

File: clusters/overlays/yukon-cars2/patches/pullsecret-sealed-patch.yaml (hunk 1 above)
Comment (novacain1) [reply clusters/overlays/yukon-cars2/patches/pullsecret-sealed-patch.yaml:15]: yep 100%, something I just added to our vault, thanks for the reminder.

**Comment by novacain1** [comment]
> I think that eventually, this should go to the `vse-clusters` repo so this repo can stay clean and serve us in multiple environments

let's talk specifics here with @jnunyez and @hhamid93 when everyone is back.

**Pull request merged**